framework/src/task_manager/task_manager_core.c

```diff
@@ -68,6 +68,19 @@ int tm_broadcast_msg[TM_BROADCAST_MSG_MAX + CONFIG_TASK_MANAGER_MAX_TASKS];
 #define TYPE_CANCEL      3
 #define TYPE_EXIT        4
 
+#define STOP_CBDATA TM_STOP_CB_INFO(handle)->cb_data
+#define STOP_CBDATA_MSG ((tm_msg_t *)TM_STOP_CB_INFO(handle)->cb_data)->msg
+#define STOP_CBDATA_SIZE ((tm_msg_t *)TM_STOP_CB_INFO(handle)->cb_data)->msg_size
+
+#define EXIT_CBDATA TM_EXIT_CB_INFO(handle)->cb_data
+#define EXIT_CBDATA_MSG ((tm_msg_t *)TM_EXIT_CB_INFO(handle)->cb_data)->msg
+#define EXIT_CBDATA_SIZE ((tm_msg_t *)TM_EXIT_CB_INFO(handle)->cb_data)->msg_size
+
+#define PARAM_CB ((tm_termination_info_t *)data)->cb
+#define PARAM_DATA ((tm_termination_info_t *)data)->cb_data
+#define PARAM_DATA_SIZE ((tm_msg_t *)((tm_termination_info_t *)data)->cb_data)->msg_size
```

Comment:
This `PARAM_DATA_SIZE` seems to mean the size of `PARAM_DATA`.
I think `PARAM_DATA_MSG_SIZE` is better.